buildroot/tests/mks_robin_pro-tests

```diff
@@ -8,6 +8,8 @@ set -e
 
 use_example_configs Mks/Robin_Pro
 opt_enable EMERGENCY_PARSER
+opt_set SERIAL_PORT 3
+opt_disable SERIAL_PORT_2
```

Comment:
If the Mks/Robin_Pro example configuration needs this change, I should do it there and revert this test.

Reply:
I already posted a PR for the examples:
https://github.com/MarlinFirmware/Configurations/pull/316

I had to work around it here for now, since I couldn't pass the CI build without the example being merged first.